content/resources/customer-experience-toolkit.md

```diff
@@ -281,14 +317,16 @@ Develop a Digital Business Strategy that embraces an improved customer experienc
     - [Finding the Best Mobile Moment for Anytime, Anywhere Government](https://digital.gov/2015/06/01/finding-the-best-mobile-moment-is-the-first-stepping-stone-to-anytime-anywhere-government/)
 - Coordinate across all channels (web, social, mobile, etc.)
     - [U.S. Digital Registry](https://digital.gov/services/u-s-digital-registry/)
+    - [Join the Social Media Community of Practice](https://digital.gov/communities/social-media/)
 
 ### Communicate clearly
 
 - [Plain Language and Writing for the Web](https://www.youtube.com/watch?v=j9FCji6-UJE) &#8211; video
+- [Join the Federal Communicators Network](https://digital.gov/communities/federal-communicators-network/)
 
 ### Adopt shared services
```

Comment:
Suggest adding a second bullet to this section: 
Customer Experience Services Evaluation and Buying Guide 
(https://www.performance.gov/cx/assets/files/CX-Evaluation-and-Buying-Guide.pdf) 
(PDF, 823 KB, 2020) - guidance on purchasing Human-Centered Design (HCD) services